components/console-backend-service/internal/domain/k8s/resourcequota_service_test.go

```diff
@@ -36,6 +36,42 @@ func TestResourceQuotaResolver_ListSuccess(t *testing.T) {
 	assert.Len(t, result, 2)
 }
 
+func TestResourceQuota_Create(t *testing.T) {
+	const (
+		namespace = "production"
+	)
+	var testCases = []struct {
+		caseName               string
+		namespace              string
+		name                   string
+		resourceLimitsMemory   string
+		resourceRequestsMemory string
+		success                bool
+	}{
+		{"Success", "production", "mem-default", "1Gi", "512Mi", true},
+		{"WithAPIGroup", "production", "mem-default", "2Gi", "3006477108", true},
+		{"duplicate", "production", "rq1", "2Gi", "3006477108", false},
+	}
+	for _, testCase := range testCases {
+		t.Run(testCase.caseName, func(t *testing.T) {
+			rq1 := fixResourceQuota("rq1", "production")
```

Comment:
Please extract this line to before `for` statement, because you always use `fixResourceQuota` with this same arguments and in each iteration you create new object. Please also do that for `58` `60` and `61` line